fixture-monkey-kotlin/src/main/kotlin/com/navercorp/fixturemonkey/kotlin/type/KotlinTypes.kt

```diff
@@ -146,3 +147,12 @@ fun Class<*>.declaredConstructor(
                 .toTypedArray(),
         )
     } ?: this.kotlin.constructors.first()
+
+fun KProperty<*>.getValueClassType(): Type {
+    val classifier = returnType.classifier
+    return if (classifier is KClass<*> && classifier.isValue) {
+        classifier.java
+    } else {
+        returnType.javaType
+    }
+}
```

Comment:
For clarity, I think it is better to inline the method.

The name `getValueClassType` confuses me as it is the method for `value class` only.

Reply:
That sounds great!
I remove getValueClassType to inline the caller method. 👍